src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -88,7 +91,10 @@ export default class DefaultBrowserBehavior implements BrowserBehavior, Extended
   }
 
   supportsCanvasCapturedStreamPlayback(): boolean {
-    return !this.isIOSSafari() && !this.isIOSChrome() && !this.isIOSFirefox();
+    return (
+      (!this.isIOSSafari() && !this.isIOSChrome() && !this.isIOSFirefox()) ||
+      parseInt(this.uaParserResult.os.version.split('.')[0]) > 15
```

Comment:
So should we be defensive on the `os.version` check here? Or is it always available? An example comment on how this looks would be helpful. Also, what does 15 indicate here? Suggest adding a static readonly variable for this to help it understand.

Probably also better to separate it its own method?

Reply:
nit: This can be a helper function `parseInt(this.uaParserResult.os.version.split('.')[0])` maybe something called `osMajorVersion`?